client/src/core/client/admin/components/ModerationReason/ModerationReason.css

```diff
@@ -0,0 +1,54 @@
+.root {
+  width: 214px;
+  padding: var(--spacing-3);
+}
+
+.innerWrapper {
```

Comment:
Is this style still used anywhere? Also, it might be nice to break out these css styles into files corresponding to the components using them for ease of finding the styles and where they're being used.

Reply:
✅